include/deal.II/simplex/quadrature_lib.h

```diff
@@ -51,6 +51,35 @@ namespace Simplex
     explicit QGauss(const unsigned int n_points_1D);
   };
 
+  /**
+   * Witherden-Vincent rules for simplex entities.
+   *
+   * Like QGauss, users should specify a number `n_points_1D` as an indication
+   * of what polynomial degree to be integrated exactly (e.g., for $n$ points,
+   * the rule can integrate polynomials of degree $2 n - 1$ exactly.). The given
+   * value for n_points_1D = 1, 2, 3, 4, 5 results in the following number of
+   * quadrature points in 2D and 3D:
+   * - 2D: 1, 6, 7, 15, 19
+   * - 3D: 1, 8, 14, 35, 59
+   *
+   * For 1D, the quadrature rule degenerates to a `QGauss<1>(n_points_1D)`.
+   *
+   * These rules match the ones listed for Witherden-Vincent in the quadpy
```

Comment:
maybe make 'quadpy' a link?

Reply:
Good idea - we can add it to the bibliography.